### DISCUSSION TITLE
A couple of questions from a newcomer

### DISCUSSION THREAD

**Discussion by tsluss** [Chia-Network/chia-blockchain#5597] [Q&A] [answered]
I've been learning how to plot and farm Chia for the last couple of weeks, and for the most part been able to get everything working and learning how to build out my farming network.  There are a couple of things that I'm still unsure of however and I was hoping for some guidance.

Ideally, I'd like to build a small cluster of inexpensive harvesters.  Maybe Raspberry Pi's or VMs for example.  I know that I can copy plots for an external storage device and farm from that but not sure if it should be copies of the plots from the full node, if they need to be moved, etc?  What I'd really like to do is create a network share and configure the harvesters to farm from that share.  I have my MacBook configured for that, and it appears to be working but I didn't notice any difference in my estimated chance for rewards.  This leads me to my next question.

Is it better to harvest different plots on different devices, have all harvesters working copies of the same plots from a storage device, or, if possible, have many harvesting across a network share?  I've tried to find these answers on my own but haven't had much luck.  What information I have found, I'm unsure if I'm just not understanding correctly of if it's not being explained very well haha.  Anyway, thanks in advance for any assistance.

**Comment by lvcivs** [comment, marked as answer]
Not sure if I understand your questions clearly. Some general points:

Don't duplicate the plots. There is no advantage in doing so, you just waste space.

If you have enough space on your network share, you can definitely store all plots there and connect your farmer to that. Just make sure that the lookup time is not too high. In that case, you don't need multiple harvesters.

The reason for multiple harvesters is usually that each machine has limited storage, so you spread out your plots over multiple machines.

**Comment by tsluss** [comment]
That's very helpful, thank you.  I've just read so many different things about different configurations, it seemed like I was going to need several computers to efficiently farm.  I'm currently running a full node with all of my storage there, starting with 14 TB, and will expand from there.  So, from what you're saying, I'm ok to run just a full node for plotting, farming, and harvesting?  Regarding, the lookup time, I'm not sure where to find that stat.  I'll dig around and see what I can find.

**Reply by lvcivs** [reply]
There is going to be a warning in the log file if your lookup time is too long. So keep an eye on that.

Most people use two machines: One for plotting (with a beefy CPU, enough RAM and SSDs for temp storage) and one for farming/harvesting (this does not need to be a powerful machine, but should rather be energy efficient and - depending on the location - quiet.) The plotter does not need to run a full node, btw, you can plot from the CLI.
